policy/modules/services/ssh.te

```diff
@@ -192,6 +192,11 @@ tunable_policy(`allow_ssh_keysign',`
 	domtrans_pattern(ssh_t, ssh_keysign_exec_t, ssh_keysign_t)
 ')
 
+ifdef(`init_systemd',`
+	systemd_user_runtime_dir_unlink(ssh_agent_tmp_t, sock_file)
```

Comment:
This should probably allow `systemd --user` to clean up any user temp file instead.